.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+
+language: cpp
+
+services:
+- docker
+
+install: skip
+
+script:
+- echo "skipping tests"
+- docker login --username ${DOCKER_USERNAME} --password ${DOCKER_PASSWORD}
+- docker pull ${DOCKER_USERNAME}/cjio:latest || true
+- docker build . --cache-from ${DOCKER_USERNAME}/cjio:latest -t ${DOCKER_USERNAME}/cjio:latest
+
+
+deploy:
+- provider: script
+  script:
+    - docker push ${DOCKER_USERNAME}/cjio:latest
+  skip_cleanup: true
+  on:
+    branch: master
+- provider: script
+  script:
+    - docker tag ${DOCKER_USERNAME}/cjio:latest ${DOCKER_USERNAME}/cjio:dev
```

Comment:
The same here.
Plus Travis is not happy about multi-line scripts in the deploy section like this one. So better to write it in a single line and use `;` to separate the commands.

Btw this is what made the val3dity deploy fail too.